### DISCUSSION TITLE
Invitations not arriving to invited poeple

### DISCUSSION THREAD

**Discussion by ferblaha** [kewisch/gdata-provider#415] [Support] [answered]
Hi,
I'm new to this addon. I'm creating a meeting in Thunderbird, and creating it in the Google Calendar. I add participants, but they never receive the invitation email.

Is there something else that needs to be configured? The event appears in my Google Calendar, but with no invited people.

Thanks.

**Comment by kewisch** [comment, marked as answer]
Please check out the options dialog, there are a few scheduling related options you need to enable. These are disabled by default because some people have been burned by invitations being sent inadvertently, e.g. when accepting an event from a different calendar in a way that Google Calendar understands this as a new event.

**Reply by stkogr** [reply]
Can you explain which options? Is it thunderbird, lightning or google? I have the issue for about 2-3 weeks now and only recognised it this week for my events created back then.

**Reply by kewisch** [reply]
In Thunderbird's add-on manager, go to the options for the Provider for Google Calendar. The first three options there should be relevant to email scheduling.
